src/amo/components/Category/index.js

```diff
@@ -271,6 +279,7 @@ export class CategoryBase extends React.Component {
             footerText={html.featuredFooterText}
             footerLink={html.featuredFooterLink}
             header={html.featuredHeader}
+            isTheme={isAddonTheme}
```

Comment:
👀

Reply:
What's been added here is code that passes the value of `isAddonTheme` into the `isTheme` prop of the `LandingAddonsCard` for each of the three instances, which have the classNames `FeaturedAddons`, `HighlyRatedAddons` and `TrendingAddons`, so this could be tested in a single test which could either check that each of those specific elements was passed the correct prop value, or could just check that all instances of `LandingAddonsCard` was passed the correct prop value. You don't need two different tests for extensions and themes, just one test that makes sure that `isTheme` on the instances of `LandingAddonsCard` is what we expect it to be.